ta/os_test_lib/sub.mk

```diff
@@ -1,6 +1,7 @@
 cppflags-$(WITH_TLS_TESTS) += -DWITH_TLS_TESTS=1
 global-incdirs-y += include
 srcs-y += os_test_lib.c
+LDFLAGS += -z noseparate-code
```

Comment:
Do we need to use this flag for all of TAs?

Reply:
Both optee-os and all TAs use this flag. 
All align with ARM.